scripts/ci/backport_audit/__main__.py

```
@@ -0,0 +1,299 @@
+                timeout=30,
+            )
+
+            for line in result.stdout.strip().splitlines():
+                if not line:
+                    continue
+
+                parts = line.split("|", 3)
+                if len(parts) != 4:
+                    raise BackportAuditError(f"Unexpected git log format: {line}")
+
+                commit_sha, author_name, author_email, subject = parts
```

Ruff (0.15.12)

[warning] Unpacked variable `author_email` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)